### ISSUE TITLE
[1.14.4] Chunks not loading

### ISSUE THREAD

**Issue by FakkaJohan** [TeamJM/journeymap#258]
JourneyMap Version: journeymap-1.14.4-5.7.0b3
Forge Version: forge-1.14.4-28.1.0

Other mods:
Worldedit (worldedit-forge-mc1.14.4-7.0.1)
Optifine (OptiFine_1.14.4_HD_U_F5)

1. I start the game with no problems
2. I select a world and load it
3. What happens next is different sometimes but allways something with chunk loading:
a. The world seems to load fine but only until a certain range and then refuses to load further. Unloading by moving out of the render distance works fine but nothing will reload.
Basically it'll load the chunks only just after entering it but after that not.
b. The world loads fine and it seems alright.
c. it does the same as in a but the range isn't square but has a weird shape, generally strange. It seems to load chunks just fine after entering the world but then stops.
d. It loads nothing at all.

It does load entities though
and journeymap functions fine, it generates the map even though I can't see it...

I don't know what is happening here...

**Comment by mysticdrew** [comment]
Remove Optifine, this is an issue caused by optifine, needs to be fixed on their end.

**Comment by FakkaJohan** [comment]
ow, i hope they will fix that soon then...

**Comment by mysticdrew** [comment]
probably not since I doubt they are working on 1.14.4 anymore.
